api/envoy/extensions/filters/http/grpc_json_reverse_transcoder/v3/transcoder.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.grpc_json_reverse_transcoder.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.grpc_json_reverse_transcoder.v3";
+option java_outer_classname = "TranscoderProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/grpc_json_reverse_transcoder/v3;grpc_json_reverse_transcoderv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: gRPC-JSON reverse transcoder]
+// gRPC-JSON reverse transcoder :ref:`configuration overview <config_http_filters_grpc_json_reverse_transcoder>`.
+// [#extension: envoy.filters.http.grpc_json_reverse_transcoder]
+
+// [#next-free-field: 6]
+// GrpcJsonReverseTranscoder is the filter configuration for the gRPC JSON
```

Comment:
it might make sense to add a note that this is the reverse of https://github.com/envoyproxy/envoy/blob/c1798d7f9cd36a27cf59ea390bdbe702aea0be3a/docs/root/configuration/http/http_filters/grpc_json_transcoder_filter.rst which allows JSON client -> gRPC service.  That way the "reverse" in the name would make more sense.

Reply:
Done